sys/include/net/dhcpv6/client.h

```diff
@@ -59,6 +59,13 @@ extern "C" {
 #define CONFIG_DHCPV6_CLIENT_PFX_LEASE_MAX (1U)
 #endif
 
+/**
+ * @brief   Maximum number of leases to be stored
+ */
+#ifndef CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX
+#define CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX (1U)
```

Comment:
Should the number of possible leases depend on `GNRC_NETIF_IPV6_DHCPV6_IA_NA_ADDR` (or vice versa)?

Reply:
I'd say vice versa. The number of non-temporary addresses is after all dependent on the number of leases you can have, not the other way around.